src/main/java/org/fundacionjala/pivotal/ui/popups/CreateProjectPopup.java

```diff
@@ -31,23 +29,30 @@ private void clickAccountDropdownList() {
         GuiInteractioner.clickWebElement(accountDropdownList);
     }
 
-    private void clickAccount1Option() {
-        GuiInteractioner.clickWebElement(account1Option);
+    private void selectAccountOption(final String account) {
+        GuiInteractioner.clickOptionFromWebElementList(
+                accountDropdownList.findElements(By.className("tc-account-selector__option-account")), account);
```

Comment:
You should have the By declared below the WebElements because you may use the by in other method in the future